mobile-browser-based-version/src/helpers/training_script/training_manager.js

```diff
@@ -49,19 +55,39 @@ export class TrainingManager {
         this.trainingInformation.learningRate
       );
     } else {
-      await trainingDistributed(
-        this.trainingInformation.modelId,
-        Xtrain,
-        ytrain,
-        this.trainingInformation.epoch,
-        this.trainingInformation.batchSize,
-        this.trainingInformation.validationSplit,
-        this.trainingInformation.modelCompileData,
-        this,
-        this.communicationManager.peerjs,
-        this.communicationManager.recvBuffer,
-        this.trainingInformation.learningRate
-      );
+      if (!trainInteroperability) {
+        this.environment.$toast.success(`Training Distributed`);
+        await trainingDistributed(
+          this.trainingInformation.modelId,
+          Xtrain,
+          ytrain,
+          this.trainingInformation.epoch,
+          this.trainingInformation.batchSize,
+          this.trainingInformation.validationSplit,
+          this.trainingInformation.modelCompileData,
+          this,
+          this.communicationManager.peerjs,
+          this.communicationManager.recvBuffer,
+          this.trainingInformation.learningRate
+        );
+      } else {
+        this.environment.$toast.success(
+          `Training Distributed and adjusting for Interoperability`
+        );
+        await trainingDistributedInteroperability(
+          this.trainingInformation.modelId,
+          Xtrain,
+          ytrain,
+          this.trainingInformation.epoch,
+          this.trainingInformation.batchSize,
+          this.trainingInformation.validationSplit,
+          this.trainingInformation.modelCompileData,
+          this,
+          this.communicationManager.peerjs,
+          this.communicationManager.recvBuffer,
+          this.trainingInformation.learningRate
+        );
+      }
```

Comment:
Maybe you can shorten this (seems repetitive):

let trainFun = trainInteroperability ? trainingDistributed : trainingDistributedInteroperability;

await trainFun(....)